components/test_raftstore/src/pd.rs

```diff
@@ -556,6 +556,7 @@ impl Cluster {
 
     fn region_heartbeat(
         &mut self,
+        _term: u64,
```

Comment:
maybe should add some logics about `term`

Reply:
I tried, but I have no idea what to add. Would you like to give some suggestions?

Reply:
Maybe like `fn handle_heartbeat_version`.